tcases-lib/src/main/java/org/cornutum/tcases/io/SystemTestDocWriter.java

```diff
@@ -147,7 +147,14 @@ protected void writeBinding( VarBinding binding)
     {
     xmlWriter_.writeTagStart( VAR_TAG);
     xmlWriter_.writeAttribute( NAME_ATR, binding.getVar());
-    xmlWriter_.writeAttribute( VALUE_ATR, binding.getValue());
+    if( binding.isValueNA())
```

Comment:
Should binding be validated here again? For Tcases as a library, maybe SystemTestDocWriter should not assume VarBinding to have been validated before? Maybe not, I find it difficult to judge.